frontend/packages/ceph-storage-plugin/src/components/block-pool/dashbaord/block-pool-dashboard-context.tsx

```diff
@@ -0,0 +1,9 @@
+import * as React from 'react';
+
+import { StoragePoolKind } from '../../../types';
+
+export const BlockPoolDashboardContext = React.createContext<BlockPoolDashboardContext>({});
+
+type BlockPoolDashboardContext = {
+  obj?: StoragePoolKind;
```

Comment:
why is this optional ?

Reply:
React.createContext<BlockPoolDashboardContext>() empty intialization thorws error. so i used ? make so it accepts "{}" empty initialization.

If I make it mandatory then I need some hack to initialize this, 

React.createContext<BlockPoolDashboardContext>({} as any)